content/lessons/js3/sublesson/objects.mdx

```diff
@@ -1936,11 +1936,10 @@ object `{}`, to store the results.
    common element.
 
 ```jsx
-;[9, 8, 7, 8, 7, 7, 7].getMostCommon()[
+;[9, 8, 7, 8, 7, 7, 7].getMostCommon()
   // returns 7 because it is the most common element
```

Comment:
Why is there a semicolon at the start of these two lines?

Reply:
@Sly, For some reason when an array is passed into a prototype function this way, it has to have a semi-colon for some reason. Otherwise, it throws an error in output. This was tested both on VScode and RunJS.
There was another issue & PR that addressed a similar issue. (PR #2205)

Reply:
@pauly48 please include this explanation in your PR description so that reviewers can understand the context behind these changes

Reply:
oh nevermind i misunderstood, my bad. All good here

Reply:
@pauly48 please wait for @SlyBouhafs to approve too, I forgot about this open conversation. Sorry @SlyBouhafs